shell/src/main/java/alluxio/cli/fsadmin/refreshworkers/StartDecommissionCommand.java

```diff
@@ -124,8 +112,7 @@ public String getCommandName() {
 
   @Override
   public String getUsage() {
-    return "start [-h] [--excluded-hosts <hostFilePath>] "
-        + "[--excluded-worker <workerHostName>]";
+    return "start [-h] [--excluded-hosts <host1> <host2>]";
```

Comment:
Would it be better to use a separator between hostnames? In the current way anything appended to the end of this command will be treated as hostnames right? That's hard to extend in the future.

Reply:
Thanks for pointing out the problem， the usage description is wrong but actually the hostnames is separated by comma, I'll fix it.